pkg/cli/sync_object_store.go

```diff
@@ -0,0 +1,115 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/minio/minio-go"
+	"github.com/spf13/cobra"
+)
+
+var srcHost string
+var srcAccessKeyID string
+var srcAccessKeySecret string
+
+var dstHost string
+var dstAccessKeyID string
+var dstAccessKeySecret string
```

Comment:
can we move these inside the NewSyncObjectStoreCmd function?

Reply:
yes